doc/architecture/adr-001.md

```diff
@@ -105,6 +110,8 @@ or operations on CosmWasm-originating digital assets.
 * The functionality in CosmWasm uploaded code may overlap with existing or future standardized native modules
 (i.e. potential extra complexity for wallets or other ecosystem tooling).
 * The attack surface is increased with the extra module.
+* "State Sync" may need to be disabled.
```

Comment:
Can "State Sync" be enabled in future versions of CosmWasm?

Reply:
it's blocked by Cosmos SDK https://github.com/cosmos/cosmos-sdk/issues/7340#issuecomment-811908201 at this moment

Reply:
wasm code are stored on 
.chain-main/wasm/wasm/wasm/  
as files

cosmwasm statesync not possible, now